serverless/src/generateColorMaps/__tests/getProjectionCapabilities.test.js

```diff
@@ -30,7 +25,6 @@ beforeEach(() => {
       debug: false
     })
 
-    // Mock the db connection
```

Comment:
Not sure if you intended to delete this comment. I think this mockKnex call is really self explanatory so its not really neded

Reply:
Did not mean to delete that line, thank you for pointing it out!